content/en/docs/setup/install/multicluster/single-network/remote/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Add a Remote Cluster
+description: Describes how to add a remote cluster to your mesh on a flat network.
+weight: 30
+keywords: [multicluster, remote-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, you can add a
+{{< gloss >}}remote cluster{{< /gloss >}} to your
+{{< gloss >}}service mesh{{< /gloss >}}.
+
+To give you clean instructions, this section starts right after you complete the
+initial configuration, and assumes that `Cluster_1` and `Cluster_2` reside on
+the same network. The new remote cluster, `Cluster_2` in this section, shares
+the {{< gloss >}}control plane{{< /gloss >}} of `CLUSTER_1`, and this guide
+assumes that `Cluster_1` and `Cluster_2` reside on the same network.
+
+Production systems use this configuration when all clusters within a region
+share a common control plane. Any configuration with one control plane per zone
```

Comment:
unfinished thought.

(and why does it have to be within a region?)

Reply:
This is explained on the deployment models concept page explaining this in a lot of detail. That page is linked across the guide multiple times. Do you want me to add a link here too?